src/Compute/v2/Models/Server.php

```diff
@@ -239,6 +239,22 @@ public function revertResize()
     }
 
     /**
+     * Gets the console output of the server.
+     *
+     * @param int $length The number of lines, by default all lines will be returned.
+     * @return string
+     */
+    public function getConsoleOutput(int $length = -1): string
```

Comment:
What is the logic behind default value`length = -1`?

Reply:
> When given a length of -1, the API will return all lines

This is an assumption thus subject to change in the future; I think we better follow official docs that `length` shall not be included in the request if we want API to return all lines.

Reply:
Good point, will make that change soon 😄.